### DISCUSSION TITLE
Container as a cargo type?

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#116] [Q&A] [answered]
I would like to be able to define "container" as a cargo type. In an intermodal terminal the actual cargo can be many different things. 
I see containers as a modern version of "frilast". 

![image](https://user-images.githubusercontent.com/57874863/131736596-2ff20f98-3f65-47e7-bf76-3ff93eece160.png)

**Comment by fjallemark** [comment, marked as answer]
A container is not a cargo type, its a kind of bearer of cargo. There are several ways to specify that cargo should go into a container:
- By selecting a container type as **Packaging Unit**/*Förpackningsenhet*.
- By selecting *containers* as **Unit of measure**/*Mängdenhet*.

Also note that in many cases container trainsets are not carrying a waybill, but runs as a *scheduled trainset* with a timetable instead of a waybill. So not all types of cargo customers need a waybill.

**Comment by AlexEhn** [comment]
I now saw that it is possible to select an "unspecified" cargo type. Perfect, then the freight customer configuration works out.

**Reply by fjallemark** [reply]
Yes, that's already used by other users for container content.
